pkg/pillar/cmd/baseosmgr/handleverifier.go

```diff
@@ -49,15 +50,85 @@ func lookupVerifierConfig(ctx *baseOsMgrContext, objType string,
 	return &config
 }
 
+// whether the object contains certs reference
+func containsCerts(safename string, ss *types.StorageConfig) bool {
+	cidx := 0
+	// count the number of cerificates in this object
+	if ss.SignatureKey != "" {
+		cidx++
+	}
+	for _, certURL := range ss.CertificateChain {
+		if certURL != "" {
```

Comment:
cen certURL be null string at all? If not, can you assert here instead of the "if" check?

if certURL == "" {
   log.Fatalf("Empty certURL - key: %s", key )
}
cidx++

Reply:
containsCerts --> This looks like it would be a good member function of ss
   ss.getCertCountsForImage(<imageSafeName>)

Reply:
Have done a refactoring to a method for Certificate Check. So that, code is common, and currently part of zedmanagertypes.go. It is called from both zedmanager and baseosmgr.